index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Microverse Day 2 Project</title>
+</head>
+<body>
+    <div class="header">
+        <h1>Hello Microverse!!!</h1>
+      </div>
```

Comment:
- [ ] To make your code more clean and readable, I think you need to mind the indentation technique you are using. This is why best practice suggested using space bar since you will always remember that your spacing in most cases requires just 2 space characters. Kindly ensure that your closing tag - in this case, your `</div>` vertically and exactly opposite the opening tag without extra spacing to overlap. This makes your code very easy to read, easy to debug, and less messy. :wink. Kindly check this :pray: